src/services/transactionService.ts

```diff
@@ -0,0 +1,134 @@
+import { DataAccessLayer } from "@blockr/blockr-data-access";
+import { Transaction } from "@blockr/blockr-models";
+import { NextFunction } from "connect";
+import { Request, Response } from "express";
+import { inject, injectable } from "inversify";
+import { HttpException } from "../utils/exceptions/httpException";
+import logger from "../utils/logger";
+
+@injectable()
+export class TransactionService {
+    protected dataAccessLayer: DataAccessLayer;
+
+    constructor(@inject(DataAccessLayer) dataAccessLayer: DataAccessLayer) {
+        this.dataAccessLayer = dataAccessLayer;
+    }
+
+    public async addTransactionAsync(request: Request, response: Response, next: NextFunction) {
+        try {
```

Comment:
Validator needs to add a transaction to the database, not the API

Reply:
Is this relevant for the current PR? Since we don't have a working validator prototype yet.